pkg/reconciler/route/resources/service_test.go

```diff
@@ -421,20 +421,21 @@ func testConfig() *config.Config {
 			TagTemplate:         network.DefaultTagTemplate,
 		},
 		Features: &apiConfig.Features{
-			MultiContainer:               apiConfig.Disabled,
-			PodSpecAffinity:              apiConfig.Disabled,
-			PodSpecFieldRef:              apiConfig.Disabled,
-			PodSpecDryRun:                apiConfig.Enabled,
-			PodSpecHostAliases:           apiConfig.Disabled,
-			PodSpecNodeSelector:          apiConfig.Disabled,
-			PodSpecTolerations:           apiConfig.Disabled,
-			PodSpecVolumesEmptyDir:       apiConfig.Disabled,
-			PodSpecPersistentVolumeClaim: apiConfig.Disabled,
-			PodSpecPersistentVolumeWrite: apiConfig.Disabled,
-			PodSpecInitContainers:        apiConfig.Disabled,
-			PodSpecPriorityClassName:     apiConfig.Disabled,
-			PodSpecSchedulerName:         apiConfig.Disabled,
-			TagHeaderBasedRouting:        apiConfig.Disabled,
+			MultiContainer:                   apiConfig.Disabled,
+			PodSpecAffinity:                  apiConfig.Disabled,
+			PodSpecTopologySpreadConstraints: apiConfig.Disabled,
```

Comment:
Is this change necessary? Curious what failed when you didn't have this?

Reply:
Done! Also deleted the line you mentioned below